analyzer/src/main/kotlin/managers/SpdxDocumentFile.kt

```diff
@@ -210,5 +211,16 @@ class SpdxDocumentFile(
     private fun getConcludedLicense(pkg: SpdxPackage): SpdxExpression? =
         pkg.licenseConcluded.takeIf { SpdxConstants.isPresent(it) }?.toSpdx()
 
-
+    /**
+     * Return a [RemoteArtifact] created with downloadLocation of the given package [SpdxPackage]
+     * if it's a file location or return en Empty RemoteArtifact.
+     * TODO: must be completed for other cases : use "sourceInfo" or "externalRefs"?
+     */
+    private fun buildSourceArtifact(pkg: SpdxPackage): RemoteArtifact {
```

Comment:
Again `build` probably is a misleading verb here, as you're not building the artifact itself. And again I'd simply propose `get` instead.